packages/form/src/components/CheckboxField/index.tsx

```diff
@@ -45,7 +47,6 @@ export const CheckboxField = <
   helper,
   tooltip,
   'data-testid': dataTestId,
-  value,
```

Comment:
You can't remove a prop like that or it will be breaking change. Instead you can mark it as deprecated using JSDoc